projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-flag/components/update-flag/update-flag.component.ts

```diff
@@ -19,25 +18,28 @@ export class UpdateFlagComponent implements OnInit {
   public errorMessages: ErrorMessage[] = [];
   public updateFlagNotEnteredErrorMessage: UpdateFlagErrorMessage = null;
   public updateFlagCharLimitErrorMessage: UpdateFlagErrorMessage = null;
-  public updateFlagHint: UpdateFlagStep;
-  public updateFlagCharLimitInfo: UpdateFlagStep;
-  public readonly updateFlagControlName = 'flagComments';
+  public updateFlagStepEnum = UpdateFlagStep;
+  public readonly caseFlagStatuses = CaseFlagStatus;
+  public readonly FLAG_COMMENTS_CONTROL_NAME = 'flagComment';
+  public readonly FLAG_STATUS_CONTROL_NAME = 'flagStatus';
+  public readonly FLAG_STATUS_CHANGE_REASON_CONTROL_NAME = 'flagStatusReasonChange';
+  public readonly FLAG_WELSH_TRANSLATION_CONTROL_NAME = 'flagWelshTranslation';
+
   private readonly commentsMaxCharLimit = 200;
 
   public ngOnInit(): void {
-    this.updateFlagHint = UpdateFlagStep.HINT_TEXT ;
-    this.updateFlagCharLimitInfo = UpdateFlagStep.CHARACTER_LIMIT_INFO;
-    this.formGroup.addControl(this.updateFlagControlName, new FormControl(''));
-
     if (this.selectedFlag && this.selectedFlag.flagDetailDisplay && this.selectedFlag.flagDetailDisplay.flagDetail) {
       const flagDetail = this.selectedFlag.flagDetailDisplay.flagDetail;
+
+      this.formGroup.addControl(this.FLAG_STATUS_CONTROL_NAME, new FormControl(flagDetail.status));
+      this.formGroup.addControl(this.FLAG_STATUS_CHANGE_REASON_CONTROL_NAME, new FormControl(''));
+      this.formGroup.addControl(this.FLAG_WELSH_TRANSLATION_CONTROL_NAME, new FormControl(false));
       // Populate flag comments text area with existing comments
-      this.formGroup.get(this.updateFlagControlName).setValue(flagDetail.flagComment);
+      this.formGroup.addControl(this.FLAG_COMMENTS_CONTROL_NAME, new FormControl(flagDetail.flagComment));
+
       if (flagDetail.name) {
         this.updateFlagTitle =
-          `${CaseFlagWizardStepTitle.UPDATE_FLAG_TITLE} "${flagDetail.name}${flagDetail.subTypeValue
-            ? `, ${flagDetail.subTypeValue}"`
-            : '"'}`;
+          `${CaseFlagWizardStepTitle.UPDATE_FLAG_TITLE} ‘${flagDetail.name}${flagDetail.subTypeValue ? `, ${flagDetail.subTypeValue}’` : '’'}`;
```

Comment:
@MateiHMCTS - it's better to use the plain ASCII `'` character rather than the curly `‘` one. The latter can cause display issues sometimes